### ISSUE TITLE
JdbcTracingUtils.decorate can set empty tag values which can break tracers

### ISSUE THREAD

**Issue by randallt** [opentracing-contrib/java-jdbc#44]
I am using the Java SpecialAgent 1.3.2 with a Wavefront Tracer. The Wavefront Tracer does not support empty tag values (an empty String) and throws an exception when processing the span. The OpenTracing spec seems to imply that empty tag values should not be used, but doesn't specifically say they are disallowed:

From https://opentracing.io/specification/ :
```
Set a Span tag
Required parameters

The tag key, which must be a string
The tag value, which must be either a string, a boolean value, or a numeric type
Note that the OpenTracing project documents certain “standard tags” that have prescribed semantic meanings.
```

The code checks for null, but allows the empty String. This should be fixed:

```
private static void decorate(Span span,
      String sql,
      ConnectionInfo connectionInfo) {
    Tags.COMPONENT.set(span, COMPONENT_NAME);
    Tags.DB_STATEMENT.set(span, sql);
    if (connectionInfo.getDbType() != null) {
      Tags.DB_TYPE.set(span, connectionInfo.getDbType());
    }
    if (connectionInfo.getDbPeer() != null) {
      PEER_ADDRESS.set(span, connectionInfo.getDbPeer());
    }
    if (connectionInfo.getDbInstance() != null) {
      Tags.DB_INSTANCE.set(span, connectionInfo.getDbInstance());
    }
    if (connectionInfo.getDbUser() != null) {
      Tags.DB_USER.set(span, connectionInfo.getDbUser());
    }
  }
```

**Comment by randallt** [comment]
Actually, it looks like it is the 'db.statement' tag that is empty in my case. Same logic applies--should not add the tag if there is no value.

**Comment by randallt** [comment]
Possible solution here:
```
private static boolean isNotEmpty(CharSequence s) {
	  return s != null && !"".contentEquals(s);
  }
  
  private static void decorate(Span span,
      String sql,
      ConnectionInfo connectionInfo) {
    
	  Tags.COMPONENT.set(span, COMPONENT_NAME);
	  if (isNotEmpty(sql)) {
		  Tags.DB_STATEMENT.set(span, sql);
	  }
	  if (isNotEmpty(connectionInfo.getDbType())) {
		  Tags.DB_TYPE.set(span, connectionInfo.getDbType());
	  }
	  if (isNotEmpty(connectionInfo.getDbPeer())) {
		  PEER_ADDRESS.set(span, connectionInfo.getDbPeer());
	  }
	  if (isNotEmpty(connectionInfo.getDbInstance())) {
		  Tags.DB_INSTANCE.set(span, connectionInfo.getDbInstance());
	  }
	  if (isNotEmpty(connectionInfo.getDbUser())) {
		  Tags.DB_USER.set(span, connectionInfo.getDbUser());
	  }
  }
```

**Comment by malafeev** [comment]
@randallt would you like to submit a PR?

**Comment by malafeev** [comment]
fixed in #46

**Comment by randallt** [comment]
Looks good. Thanks.
